cypress/e2e/umd.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('web client recording spec', () => {
+	;['unpkg', 'jsdelivr'].map((source) => {
```

Comment:
Is this `;` necessary?

Reply:
eslint puts this here for some reason when i inline the array. i'll bring it out to clean this up.